taf/src/main/java/com/epam/gomel/tat/test/YandexDiskCreateTextDocumentTest.java

```diff
@@ -0,0 +1,114 @@
+package com.epam.gomel.tat.test;
+
+import com.epam.gomel.tat.page.YandexDiskHomePage;
+import com.epam.gomel.tat.page.menu.YandexDiskMenuOfFilesPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import static org.testng.Assert.assertTrue;
+
+public class YandexDiskCreateTextDocumentTest {
+
+    private static final String LOGIN = "andrew-394";
+    private static final String PASSWORD = "andrew394";
```

Comment:
Do login/password public and move in an appropriate place (I suppose right now it is AbstractScreen)